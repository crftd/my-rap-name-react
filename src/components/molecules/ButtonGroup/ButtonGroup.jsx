import React from 'react';
import styled from 'styled-components';

const ButtonGroupWrapper = styled.div`
  display: flex;
  justify-content: center;
`;

export default ({ children }) => <ButtonGroupWrapper>{children}</ButtonGroupWrapper>;
