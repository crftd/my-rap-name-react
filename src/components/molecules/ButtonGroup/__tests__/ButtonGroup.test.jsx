import React from 'react';
import { shallow } from 'enzyme';
import ButtonGroup from '../ButtonGroup.jsx';

describe('[Snapshot] ButtonGroup', () => {
  it('should render', () => {
    // Arrange

    // Act
    const actualTree = shallow(<ButtonGroup />);
    // Assert
    expect(actualTree).toMatchSnapshot();
  });
});
