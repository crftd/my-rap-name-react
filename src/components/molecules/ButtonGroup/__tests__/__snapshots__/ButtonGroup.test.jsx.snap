// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Snapshot] ButtonGroup should render 1`] = `<styled.div />`;
